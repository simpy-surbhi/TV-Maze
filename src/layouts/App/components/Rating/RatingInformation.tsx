import { Typography } from "@material-ui/core";
import * as React from "react";
import { useStyles } from "./styles";
import { Rating } from "@material-ui/lab";

interface Props {
  ratingValue: string;
}

export const RatingInformation: React.FC<Props> = ({ ratingValue }) => {
  const classes = useStyles();
  return (
    <React.Fragment>
      <div className={classes.ratingText}>
        <Rating defaultValue={1} precision={1} max={1} readOnly size="large" />
        <Typography variant="h6">{ratingValue} </Typography>
        <Typography variant="subtitle2">/10</Typography>
      </div>
    </React.Fragment>
  );
};
