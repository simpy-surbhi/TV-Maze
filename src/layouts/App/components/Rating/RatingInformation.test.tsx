import React from "react";
import { render } from "@testing-library/react";
import { RatingInformation } from "./RatingInformation";

test("RatingInformation renders correctly", () => {
  const { container } = render(<RatingInformation ratingValue="2.2" />);
  expect(container.firstChild).toMatchSnapshot();
});
