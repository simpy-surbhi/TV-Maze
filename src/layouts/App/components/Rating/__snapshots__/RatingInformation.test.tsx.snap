// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingInformation renders correctly 1`] = `
<div
  class="makeStyles-ratingText-1"
>
  <span
    aria-label="1 Star"
    class="MuiRating-root MuiRating-sizeLarge MuiRating-readOnly"
    role="img"
  >
    <span>
      <span
        class="MuiRating-icon MuiRating-iconFilled"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
          />
        </svg>
      </span>
    </span>
  </span>
  <h6
    class="MuiTypography-root MuiTypography-h6"
  >
    2.2
     
  </h6>
  <h6
    class="MuiTypography-root MuiTypography-subtitle2"
  >
    /10
  </h6>
</div>
`;
